static/js/selectors/websites.ts

```diff
@@ -26,3 +27,15 @@ export const getWebsiteCollaboratorDetailCursor = createSelector(
       find(propEq("username", username), collaborators[name])
     )
 )
+
+export const getWebsiteContentListingCursor = createSelector(
+  (state: ReduxState) => state.entities?.websiteContentListing ?? {},
+  listing => (name: string, type: string) =>
+    // not memoized since there are two arguments
```

Comment:
it's possible to still memoize with two arguments, you just need to supply a custom function to determine how to memoize (i.e. how to convert arguments into a key for the cache the `memoize` function uses).

there's an example here in Open: https://github.com/mitodl/open-discussions/blob/master/static/js/lib/queries/learning_resources.js#L187